guides/static/guides/css/guide_list.css

```diff
@@ -14,3 +14,15 @@
 .no-underline {
     text-decoration: none;
 }
+
+#search-button{
+    position: relative;
+    right: -535px;
+    top: -40px;
+    color: white;
+    background-color: #1c86f2;
```

Comment:
This doesn't display on Safari for some reason, the button is white:
![image](https://user-images.githubusercontent.com/22679924/41378469-6cc052fe-6f5f-11e8-89be-82c43d9a7a06.png)
On Firefox, it works fine though.

Reply:
That's weird, on my Safari it's completely fine.
![screen shot 2018-06-13 at 22 54 23](https://user-images.githubusercontent.com/31344273/41380423-cc5b932a-6f5c-11e8-991c-8dc26f72431f.png)

Reply:
My bad. I forgot that I still had the old CSS cached. Works fine now 👍